libraries/provision/ansible/playbooks/start-sgload.yml

```diff
@@ -0,0 +1,82 @@
+---
+- hosts: load_generators
+  any_errors_fatal: true
+
+  # Kill gateload
+  tasks:
+  - name: kill previous sgload screen
```

Comment:
I know @tleyden has expressed dissatisfaction with using screen to run this, can you think of any alternatives?